gui/next/src/lib/database/Restore.svelte

```diff
@@ -0,0 +1,47 @@
+<!-- ================================================================== -->
+<script>
+
+// imports
+// ------------------------------------------------------------------------
+import { page } from '$app/stores';
+import { record } from '$lib/api/record';
+import { state } from '$lib/state';
+
+import Icon from '$lib/ui/Icon.svelte';
+
+
+export let table;
+export let id;
+
+let form;
+
+const remove = async (event) => {
```

Comment:
@z-x should it be `restore` or something like that?

Reply:
Yes it should, thanks!